Corona-Warn-App/src/main/res/layout/view_consent_status.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:focusable="true">
+
+    <View
+        android:id="@+id/test_result_illustration_divider"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/card_divider"
+        android:background="@color/colorHairline"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"/>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/test_result_subtitle"
+        android:layout_width="@dimen/match_constraint"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/spacing_tiny"
+        android:layout_marginTop="@dimen/spacing_small"
+        android:focusable="true"
+        app:layout_constraintEnd_toEndOf="@+id/guideline_end"
+        app:layout_constraintStart_toStartOf="@+id/guideline_start"
+        app:layout_constraintTop_toBottomOf="@+id/test_result_illustration_divider">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/test_result_subtitle_icon"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/spacing_mega_tiny"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintVertical_bias="0.0">
+
+            <ImageView
+                style="@style/icon"
+                android:layout_width="@dimen/icon_size_risk_details_behavior"
```

Comment:
This custom view has no relationship with risk details behavior. I would prefer if we either add a new dimension (if there are other similar views with such icons as this one, or just hardcode it here as it a custom view.